pkgs/top-level/python-packages.nix

```diff
@@ -7126,6 +7126,30 @@ in {
     };
   };
 
+  grammalecte = buildPythonPackage rec {
```

Comment:
If it is a python library then you can keep it here. However the expression to `python-modules` as described in the header of this file.